DictionaryTags.txt

```diff
@@ -270,7 +270,7 @@ ch_pos			255     ch_pos_rec	-	"Channel position"
 ch_unit			256	enum(unit)	-	"Unit of the data"
 ch_unit_mul		257	int32		rel	"Unit multiplier exponent.
 ch_dacq_name		258	string		-	"Name of the channel in the data acquisition system. Corresponds to fiffChInfoRec.name."
-#<reserved>		259
+ch_coil_type		259	int32		-	"Coil type in coil_def.dat"
```

Comment:
This was one of two entries that is in the `fiffChInfoRec` but not in the spec

Reply:
The #<reserved> tags are reserved. In practice that means that somewhere in this universe there are files that are using the tags, but they are for some reason withdrawn or not documented. So lets avoid them.